Python/Product/EnvironmentsList/EnvironmentPathsExtension.xaml

```diff
@@ -13,13 +14,16 @@
         <CommandBinding Command="{x:Static l:EnvironmentPathsExtension.OpenInFileExplorer}"
                         CanExecute="OpenInFileExplorer_CanExecute"
                         Executed="OpenInFileExplorer_Executed" />
+        <CommandBinding Command="{x:Static l:EnvironmentPathsExtension.OpenInBrowser}"
+                                                            CanExecute="OpenInBrowser_CanExecute"
+                                                            Executed="OpenInBrowser_Executed" />
     </UserControl.CommandBindings>
     <UserControl.Resources>
         <ResourceDictionary>
             <ResourceDictionary.MergedDictionaries>
                 <ResourceDictionary Source="pack://application:,,,/Microsoft.PythonTools.EnvironmentsList;component/EnvironmentView.xaml" />
                 <ResourceDictionary Source="pack://application:,,,/Microsoft.PythonTools.EnvironmentsList;component/FileNameLabel.xaml" />
-
+                <ResourceDictionary Source="pack://application:,,,/Microsoft.PythonTools;component/PythonTools/Wpf/ModernStyles.xaml" />
```

Comment:
this defines a bunch of styles on common controls and is not meant to be used in conjunction with controls.xaml, which does the same thing but is the older version used in tool windows and older dialogs (EnvironmentView.xaml imports controls.xaml)

can you do without it?

Reply:
i dont think the hyperlink style was being applied without it. but maybe I can pull in just the hyperlink style

Reply:
for the issue with mergeddictionaries and ModernStyles.xaml I could:
 
a) Add ModernStyles.xaml to the top of the list that way controls.xaml would override anything defined in ModernStyles.xaml
 
b) move hyperlinkbutton to its own style file

Reply:
apparently the order is the last resource is taken in mergeddictionaries.  https://docs.microsoft.com/en-us/dotnet/framework/wpf/advanced/merged-resource-dictionaries?redirectedfrom=MSDN